CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to the vscode-opa extension will be documented in this file.
```

Comment:
While you're here, I could imagine us dropping this line.

Reply:
Haha! Well, starting from now they will be :)